src/components/HorizontalStepper/HorizontalStepper.tsx

```diff
@@ -0,0 +1,82 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './HorizontalStepper.module.css';
+import HorizontalStep from '../HorizontalStep';
+
+export interface Props {
+  /**
+   * Zero-based index that identifies which index is the active step.
+   */
+  activeIndex: number;
```

Comment:
I didn't realize until I read the warning code below that if all the steps are completed, the `activeIndex` is actually a step that doesn't exist, one beyond the last step – is that right? If there are 5 steps and you've completed all of them, the `activeIndex` would be 5, a non-existent step? That seems a bit odd to me.. I wonder if `activeIndex` should be `null` if the steps are completed since there is no active step.

Reply:
I do agree that it's a little awkward to have the `activeIndex` actually point to a nonexisting index on the `steps` array, but I went with this since I see the parent logic being written like:
```
  const onClickBack = () => {
    if (activeIndex > 0) setActiveIndex(activeIndex - 1);
  };
  const onClickNext = () => {
    if (activeIndex < steps.length) setActiveIndex(activeIndex + 1);
  };
```
How do you feel about this?

Reply:
Yeah I'm not sure. I can see how it's adding extra logic for them to check if it's the last step and if so set it to `null`, but I also feel like the one-step-past-the-edge API is odd. Maybe just add a comment to the prop explaining what the `activeNumber` should be if all the steps are completed.

Reply:
Sounds good!